packages/es-components/src/components/containers/popover/Popup.js

```diff
@@ -81,7 +238,7 @@ Popup.propTypes = {
   trigger: PropTypes.node,
   children: PropTypes.node,
   onHide: PropTypes.func,
-  placement: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
```

Comment:
This shouldn't qualify as a major breaking change, since we don't export `Popup`, right? The external API remains the same, as you mentioned.

Reply:
I don't think we export `Popup` from the index file, no, and it's only used in `Popover` as far as I know. I considered it a major change because I'm not 100% sure it won't break anything, but ideally it shouldn't.